tpmtool.cpp

```diff
@@ -856,9 +977,9 @@ main (
     //
     // Check for the special enumerate option with no other arguments
     //
-    if (strcmp(Arguments[1], "-e") == 0)
+    if ((strcmp(Arguments[1], "-e") == 0) || (strcmp(Arguments[1], "-qa") == 0))
```

Comment:
Just do two strcmps -- one for -e, and one for -qa, then pass in true/false to enumeratespaces.